src/fastmcp/server/auth/models.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+from mcp.server.auth.provider import AccessToken
+
+
+class AccessTokenWithClaims(AccessToken):
```

Comment:
Please move this class into src/server/auth/auth.py rather than a new file. 

I'm also ok (even encouraging) if you want to name this class `AccessToken` since it will be the canonical access token class for the FastMCP framework, and subclass `from mcp.server.auth.provider import AccessToken as _SDKAccessToken` or similar to avoid name conflicts

Reply:
Thanks the comments are addressed, including moving, renaming, and updating `dependencies.py`! I had to add some more logic to `get_access_token()` to satisfy the linter and ensure type safety.